server/plugins/index.js

```diff
@@ -43,6 +43,20 @@ function registerPlugins (server, config, callback) {
       }
     })
 
+  var thirdPartyPlugins = require('hoodie.plugins')
+
+  $.each(thirdPartyPlugins, function(){
+    concat(
+    path.resolve('hoodie/server').filter(checkModule)
+    )
+    .map(function (register) {
+      return {
+        options: options,
+        register : require(register)
+      }
+    })
+  })
```

Comment:
see my comments above, it is similar with the server-side plugins. You can find out if a server extension exists in a 3rd party plugin using `checkModule('name-of-the-npm-package/hoodie/server')`. If it returns true, add it to the `thirdPartyPlugins` array and then pass it all together to `server.register` with `hapiPlugins.concat(thirdPartyPlugins).concat(localPlugins)`